docs/ref/models/expressions.txt

```diff
@@ -252,6 +252,16 @@ The ``Func`` API is as follows:
         A class attribute that denotes the character used to join the list of
         ``expressions`` together. Defaults to ``', '``.
 
+    .. attribute:: arity
+
+        .. versionadded:: 1.10
+
+        A class attribute that denotes the exact number of expressions
+        expected by the function. If this attribute is set and the function
```

Comment:
"the number of arguments" or "the number of expression arguments" would fit better I think.